package.json

```diff
@@ -4,10 +4,12 @@
   "repository": "balancer-labs/snapshot-hub",
   "license": "MIT",
   "scripts": {
-    "lint": "vue-cli-service lint ./server/**/*.ts",
-    "express": "vue-cli-service express:watch",
-    "express:run": "vue-cli-service express:run",
-    "start": "vue-cli-service express:run"
+    "lint": "eslint ./server/**/*.ts --fix",
```

Comment:
This script dont work for me, it does for you?

Reply:
ya working for me.. you are getting some error?

Reply:
I mean, i introduce some lint error and run the script but nothing got fixed. This doesn't happen for you?

Reply:
@bonustrack fixed it .. please review now